tutorials/R/neonUtilities/neonUtilities.Rmd

```diff
@@ -319,6 +377,57 @@ NEON_D01_HOPB_DP3_716000_4705000_CHM.tif downloaded to /Users/neon/DP3.30015.001
 The files should now be downloaded to a new folder in your 
 working directory.
 
+## Download remote sensing files for specific coordinates: byTileAOP()
+
+Often when using remote sensing data, we only want data covering a
+certain area - usually the area where we have coordinated ground 
+sampling. `byTileAOP()` queries for data tiles containing a 
+specified list of coordinates. It only works for the tiled versions 
+of the remote sensing data, i.e. the ones with data product IDs 
+beginning with "DP3".
+
+Here, we'll download tiles of vegetation indices data (DP3.30026.001) 
+corresponding to select observational sampling plots. For more information 
+about NEON spatial data, see the 
+<a href="https://www.neonscience.org/neon-api-usage" target="_blank">
+API tutorial</a> and the in-development <a href="https://github.com/NEONScience/NEON-geolocation/tree/master/geoNEON" 
+target="_blank">
+geoNEON package</a>.
+
+For now, assume we've used the API to look up the plot centroids of 
+plots SOAP_009 and SOAP_011 at the Soaproot Saddle site.
+The coordinates of the two plots in UTMs are 298755,4101405 and 
+299296,4101461. These are 40x40m plots, so in looking for tiles 
+that contain the plots, we want to include a 20m buffer.
```

Comment:
Given our plots are square, the corner is actually 28.28m from the centroid, correct.  Better to use this as the buffer, or just a round 30m to make sure to capture all tiles the overlap the plot?

Reply:
Making a round buffer was way too annoying when I wrote this code, it's a square buffer :)